### DISCUSSION TITLE
Where can i get your Obsidian CSS?

### DISCUSSION THREAD

**Discussion by tallguyjenks** [BryanJenksCommunity/FAQ#2] [Q&A] [answered]
### Question

Where can I get your Obsidian CSS?

**[>> Download Here <<](https://bryanjenkstech.ck.page/d4ec0713d5)**

**Comment by tallguyjenks** [comment, marked as answer]
Converted from issue

**Reply by mnott** [reply]
Thanks! How do use those assets?

**Reply by tallguyjenks** [reply]
for the CSS theme you would put it into the hidden directory `.obsidian/theme` inside your obsidian vault
